packages/cronie/build.sh

```diff
@@ -4,7 +4,7 @@ TERMUX_PKG_LICENSE="ISC, BSD 2-Clause, BSD 3-Clause, GPL-2.0, LGPL-2.1"
 TERMUX_PKG_LICENSE_FILE="COPYING, COPYING.obstack"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1.7.2"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_REVISION=3
```

Comment:
`TERMUX_PKG_REVISION` is a counter for the published package revisions.
It only needs to be incremented once per pull request and only takes effect when the package is uploaded to the repositories, so this should stay at `2` and would only be incremented to `3` if another revision to the `cronie` package is made after this one is merged.
```suggestion
TERMUX_PKG_REVISION=2
```